src/app/header/header.component.ts

```diff
@@ -50,8 +54,8 @@ export class HeaderComponent implements OnInit {
 
   getTogglePath(): void {
     this.toggleService
-        .getToggles()
-        .then(togglePaths => this.togglePaths = togglePaths);
+      .getToggles()
```

Comment:
It is acceptable to indent to the "dot" for this type of notation. It is common practice. It doesn't break anything but most find it easier to read.